<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>668</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Variable</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>set?</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Setter Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Setter Specification</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Comment</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTask_Chooser_Frontend">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_json"/>
    <addaction name="save_json"/>
    <addaction name="separator"/>
    <addaction name="import_tb"/>
    <addaction name="import_grc"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuTask_Chooser_Frontend"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_json">
   <property name="text">
    <string>Open Task JSON..</string>
   </property>
  </action>
  <action name="save_json">
   <property name="text">
    <string>Save Task JSON...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="import_tb">
   <property name="text">
    <string>Import top_block from Python module...</string>
   </property>
  </action>
  <action name="import_grc">
   <property name="text">
    <string>Import GNU Radio companion Flow Graph</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
